tools/static-assets/skel/.meteor/packages

```diff
@@ -19,3 +19,4 @@ ecmascript              # Enable ECMAScript2015+ syntax in app code
 
 autopublish             # Publish all data to the clients (for prototyping)
 insecure                # Allow all DB writes from clients (for prototyping)
+census                  # Track anonymous info to help us improve Meteor
```

Comment:
First, unless I’m wrong reading the code, I wouldn’t qualify an IP address nor the application URL as some “anonymous info”.

Furthermore I believe that adding a tracking package by default is a very bad signal sent to both the Meteor and the broader JavaScript communities. We don’t expect jQuery to send data to “help the jQuery company”, nor React to send data to Facebook, nor Meteor to send data to MDG. At the very least, if you have a real need for this data, let the developers opt-in – instead of giving them a way to opt-out.

My intuition is that built-in tracking code has a risk to deteriorate Meteor reputation. Maybe we should discuss this issue on the forum?